src/HDF5.jl

```diff
@@ -589,22 +590,27 @@ See also
 """
 function create_dataset(
```

Comment:
Technically this is a new convention on styling the breaking long method signatures in the repo, but it's not that important.  We usually would align the second lines, similar to the majority of the julialang repo (I think).

Reply:
I'm using https://github.com/invenia/BlueStyle#method-definitions as a style reference. In particular, I'm using the format they have here:

```julia
# Yes:
function foobar(
    df::DataFrame,
    id::Symbol,
    variable::Symbol,
    value::AbstractString,
    prefix::AbstractString="",
)
    # code
end
```

I'm not sure if anyone else has tried to codify a style for Julia. `JuliaLang/Julia` is not very consistent stylistically, but I think Blue extracts the best practices.

Reply:
I'm not sure where to increase the ident to. Identing it more seems to contradict this part of the Blue style guide.
```julia
# No: Indented too much.
function foobar(
        df::DataFrame,
        id::Symbol,
        variable::Symbol,
        value::AbstractString,
        prefix::AbstractString="",
    )
    # code
end
```

Reply:
We don't try to follow the Blue style guide and neither does the main Julia repo. So I wouldn't feel constrained by that guide. After all, it's just one convection.

Reply:
Feel free to suggest an indentation.

Reply:
I suggest we don't do anything and keep it as is for now.